roles/base/tasks/redhat_tasks.yml

```diff
@@ -10,6 +10,7 @@
     update_cache: true
     state: latest
   with_items:
+    - gcc
```

Comment:
Oh, ok! I will try, thanks.

-Erik

On 27 Apr 2016, at 14:39, Madhav Puri wrote:

>> @@ -10,6 +10,7 @@
>>      update_cache: true
>>      state: latest
>>    with_items:
>> +    - gcc
>
> @erikh is this due the issue that we saw with gcc not being packaged 
> in packer boxes? It should be fixed in new boxes 
> (https://github.com/contiv/build/pull/60), can you try the new box 
> `0.3.6ba5a75` ?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/180/files/9010d5bfb79e5fff3d0a25132af6edf21a833ff5#r61341337